storage/src/tests/persistence/persistencetestutils.h

```diff
@@ -35,10 +35,9 @@ struct PersistenceTestEnvironment {
 };
 
 class PersistenceTestUtils : public CppUnit::TestFixture {
-private:
+public:
```

Comment:
Intentional to make `_env` public? This is just a test fixture, so in that sense there shouldn't be a problem with it.

Reply:
It is required for the upcoming PR. I just took out the whole diff and sorted out the ones that had no semantic issues to reduce the complexity.